csp/middleware.py

```diff
@@ -50,35 +48,48 @@ def process_response(self, request, response):
             return response
 
         # Check for ignored path prefix.
-        prefixes = getattr(settings, 'CSP_EXCLUDE_URL_PREFIXES', ())
+        # TODO: Legacy setting
```

Comment:
Design Decision: how we want to handle CSP_EXCLUDE_URL_PREFIXES under the new settings dict (could punt on it and just leave the legacy setting).

Reply:
This should be easy enough to fold in.